api/allIncidents/incidentsModel.js

```diff
@@ -114,7 +113,12 @@ function getLastID() {
  * Ideally google maps API can take city/state and populate lat/long on backend.
  */
 async function createIncident(incident) {
+  const newMax = await db('force_ranks').max('incident_id').first();
+  const { max } = newMax;
+  const newIncidentId = max + 1;
```

Comment:
I'm curious - how did we confirm that the deployed API & database aren't automatically incrementing each incident's ID? I ask because the migration that creates the table we're using is clearly set up to autoincrement these IDs, as seen in this screenshot on line 5.

![image](https://user-images.githubusercontent.com/55460082/134443373-c9b84b14-a431-4095-a1e1-b51b40089d34.png)

Reply:
Isn't the backend using data science's database in production?

Reply:
Yes it is, let me know if you need any more info after our most recent discussion in the standup @paulstgermain